src/shared/components/person/settings.tsx

```diff
@@ -595,6 +617,53 @@ export class Settings extends Component<any, SettingsState> {
     );
   }
 
+  importExport() {
+    return (
+      <>
+        <h2 className="h5">Import/Export</h2>
+        <p>Import and export your account settings as JSON.</p>
```

Comment:
i18n all these strings.

Reply:
Of course. Just want to make sure the text works in context before creating a PR for the translations.